### ISSUE TITLE
Could not use ** for jsonpath in java

### ISSUE THREAD

**Issue by eakarsu** [rest-assured/rest-assured#753]
I would like to fetch all embedded json elements but I could not make it working. Let's say I would like to filter all b's here. What gpath syntax to filter all b's here?
{
  a:{
  c: {
     b:1
  }
  b:45
 d: {
   b:444
}
}

**Comment by johanhaleby** [comment]
Good question, not quite sure how you get "b" in an arbitrary depth in JSON using Gpath. Perhaps you can ask on stackoverflow? I would be interested to know this as well.

**Comment by eakarsu** [comment]
Jsonpath has syntax "$..b" that would select all b's. I don't know why gpath does not have it.
But gpath for XML has it as far as i can see from examples

**Comment by johanhaleby** [comment]
Yes for XML this would work with gpath as well, but it doesn't work with json afaik (and I don't know why).

**Comment by johanhaleby** [comment]
And just to make it clear, GPath is a part of Groovy so there's little I can do (I think) to improve it.

**Comment by yuyu1987** [comment]
It’s frustrating not to support depth
